src/frontend/screens/Game/GamePage/index.css

```diff
@@ -505,8 +505,12 @@
         color: var(--navbar-active);
       }
 
+      &:first-child {
+        margin-top: -0.1rem;
+      }
+
       &:last-child {
-        padding-bottom: 0;
+        margin-bottom: -0.1 rem;
```

Comment:
there's a broken syntax here, the space `-0.1 rem` instead of `-0.1rem`, but I'd rather set `0` instead of negative margin